tasks/eth/031-uni-holocene-systemconfig-update/README.md

```diff
@@ -0,0 +1,64 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+Upgrades the **Unichain Mainnet** Fault Proof PDG to a new challenger and upgrades the System Config to the Holocene version.
```

Comment:
Can we please double check this. From the state diff, I don;'t think we are are doing anything with the fault proofs system with this task.